presto-spark-classloader-interface/src/main/java/com/facebook/presto/spark/classloader_interface/PrestoSparkDiskPage.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spark.classloader_interface;
+
+import java.io.Serializable;
+import java.net.URI;
+import java.util.List;
+
+public class PrestoSparkDiskPage
+        implements Serializable, PrestoSparkTaskOutput
+{
+    private long broadcastId;
```

Comment:
Instead of storing broadcastId in every output handle i would suggest refactor it a little bit.

Let's add a wrapper class:

```
class PrestoSparkBroadcastDescriptor {
  private final long broadcastId;
  private final List<PrestoSparkStorageHandle> handles;
  ...
}
```

Then instead of creating a `Broadcast<List< PrestoSparkDiskPage >>` create a `Broadcast< PrestoSparkDiskPage>`

What do you think?